src/MoBi.Presentation/Extensions/BreadCrumbExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MoBi.Presentation.DTO;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MoBi.Presentation.Extensions
+{
+   public static class BreadCrumbsExtensions
+   {
+      public static bool HasAtLeastTwoDistinctValues(this IEnumerable<IBreadCrumbsDTO> breadcrumbs, int pathElementIndex)
```

Comment:
do we need the IBreadCrumbsDTO (the interface). Can this be removed too? (not sure)